packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -1,26 +1,34 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { isMobile } from '@deriv/shared';
-import Dropdown from '../dropdown/dropdown.jsx';
-import SelectNative from '../select-native/select-native';
+import Dropdown from '../dropdown/index.js';
```

Comment:
changing this doesnt have any effect on component style?

Reply:
asking this because normally index files have styles included too

Reply:
Yes but when i am using dropdown.jsx it was throwing "that there was no default export".That is why i had to change it to index.js

Reply:
so why dont you import it like a named component?
import { DropDown } from '....'